### PR TITLE
Pr/dialog text

### PR DESCRIPTION
**Pull request by uli42**

### CODE REVIEW

File: nxcomp/src/Children.cpp [hunk 1]
```diff
@@ -274,11 +275,17 @@ int NXTransDialog(const char *caption, const char *message,
...
 
-    if (i == 0)
+    if (i == 0 || i == 2)
     {
-
       strcpy(command, "nxclient");
+    }
+    else if (i == 1 || i == 3)
+    {
+      strcpy(command, "nxdialogx");
```
Comment (sunweaver) [inline nxcomp/src/Children.cpp]: s/nxdialogx/nxdialog/ ?

**Review by sunweaver — CHANGES_REQUESTED** [review]
Please fix the typo I spotted. Except from that, this is ready to merge.

**Comment by sunweaver** [comment]
@uli42: while at it, please rebase again 3.6.x. Thanks.

**Comment by uli42** [comment]
done

**Pull request merged**